02-deeper-dive/02-03_optimization/02-03-04_data.md

```diff
@@ -6,3 +6,7 @@ Data is the fundamental “stuff” you feed into your optimization model, the i
 In the financial world, data is the assets you can buy or sell and their prices. In the aerospace world, data is the unit weight and cost of carrying a certain kind of fuel. In design, data is the density or price of construction materials, how many hours of sunlight a room can expect to receive in a day, or any goals that are important to your design exploration that you can define mathematically. The optimization model takes all this data and works to maximize or minimize values as specified. 
 
 Real-world optimization problems are invariably solved algorithmically and there are often many algorithms to solve the same problem (several of which will be introduced below). The most common algorithm used in generative design for architectural and engineering workflows is the genetic algorithm.
+
+Data and how data is organized would determine how easy it would be to navigate through solutions and find optimal solutions. 
```

Comment:
```suggestion
The raw data (whether input data for generation or the output of the generative process) and how it is organized will determine how easy it will be to navigate through solutions and find the most suitable (optimal) one. 
```